packages/manager/modules/sms/src/sms/compose/telecom-sms-sms-compose.controller.js

```diff
@@ -479,8 +480,44 @@ export default class {
           validReceivers: size(validReceivers),
         },
       });
+
       this.resetForm(form);
-      this.TucToast.success(this.$translate.instant('sms_sms_compose_status_success'));
+      if (!_.isEmpty(invalidReceivers) && _.isEmpty(validReceivers)) {
+        const receiverNumber = (invalidReceivers.length > 1)
```

Comment:
You should have a look at: https://angular-translate.github.io/docs/#/guide/14_pluralization

Reply:
Good idea @jleveugle ! It's exactly what I want to do.

Reply:
I'll try to use it, but it's like the mapping with message pluralized is not set. I applied the recommandation in the documentation but it doesn't work.